src/side-navigation/side-navigation.scss

```diff
@@ -170,3 +171,41 @@
     }
   }
 }
+
+@mixin iui-side-navigation-submenu {
+  display: flex;
+  flex: 0 0 auto;
```

Comment:
Is it needed?

Reply:
With current html demo, there is no need of flex at all in here...

Reply:
Removed the flex parts.